### ISSUE TITLE
getStepsCount() & getDailyDistanceSamples() always returns no data.

### ISSUE THREAD

**Issue by nehvaleem** [StasDoskalenko/react-native-google-fit#13]
I've got an issue with getting any steps from my account. I've authorized google fit, added signatures to firebase etc. but when I try to get steps using one of provided methods I am getting empty array or message that there is no steps data available at current time (event when i try to set time for last week or more). I've checked my fit data via web (http://fit.google.com) and there are plenty of samples, but I can't figure out what am I doing wrong.

any help maybe? I am using google play services 9.8.0, tested on physical device running on android 6.0.

**Comment by HilSny** [comment]
This is getting steps directly from the app on your phone and not the cloud at fit.google.com. Do you have the app installed on your test device?

**Comment by nehvaleem** [comment]
yeah. i do and there are also steps. Is there any way to obtain cloud data by the way?

**Comment by HilSny** [comment]
We used the REST API to do that previously, you'll need to make sure you're authorized by google to use it. See here: https://developers.google.com/fit/rest/
